server.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const db = require('./data/db');
+
+const port = 5555;
+const server = express();
+server.use(express.json());
+
+server.get('/', (req, res) => {
+    //1st arg: route where a resource can be interacted with
+    //2nd arg: cb to deal with sending res and handling incoming.
+    res.send('Hello from express');
+});
+
+server.post('/api/users', (req, res) => {
+    const { name, bio } = req.body;
+    db
+        .insert({ name, bio })
```

Comment:
Great start here! Consider implementing a check to make sure that both `name` and `bio` values exist. From the readme:

> If the request body is missing the name or bio property:
> cancel the request.
> respond with HTTP status code 400 (Bad Request).
> return the following JSON response: { errorMessage: "Please provide name and bio for the user." }.